RxSwiftWorkshop/Flights/ViewModel/FlightsViewModelSpec.swift

```diff
@@ -0,0 +1,91 @@
+//
+// Created by Maciej Oczko on 02.05.2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import Quick
+import Nimble
+import RxSwift
+import RxTest
+
+@testable import RxSwiftWorkshop
+
+final class FlightsViewModelSpec: QuickSpec {
+    override func spec() {
+        describe("FlightsService") {
+
+            var sut: FlightsViewModel!
+            var schipolService: FakeSchipolService!
+            var iataService: FakeIATAService!
+
+            var scheduler: TestScheduler!
```

Comment:
Och, I see. The rule is very good.

However in specs it's a little bit different. Here the order of blocks (so initializations and deinits) is important. So having properties in memory for all time of tests being executes might be dangerous.

I'd rather go with optionals for every thing other that system under test (sut). Anyway, the sooner test fails the better, so all in all, IUOs here are not so bad.